projects/tags-input-demo/src/app/app.component.ts

```diff
@@ -49,6 +50,24 @@ export class AppComponent {
     ddHasBackdrop: false
   };
 
+
+  ImageTagsInputConfig: AngularTagsInputConfig = {
+    showTagsSelectedInDD: true,
+    hideAddedTags: false,
+    hideDDOnBlur: true,
+    nestedTagProperty: 'subOrdinates',
+    onlyFromAutoComplete: true,
+    additionalClasses: 'ti-tags-input',
+    displayProperty: 'full_name',
+    identifier: 'Id',
+    toggleSelectionOnClick: true,
+    nestedTagParentProp: 'parent_id',
```

Comment:
`nestedTagParentProp` and some other props aren't being used for this example. Please remove them as before.

Reply:
fixed